apps/storybook.namekit.io/stories/Namekit/Input.stories.tsx

```diff
@@ -112,6 +112,17 @@ export const WithSlotRight: Story = {
   },
 };
 
+export const WithError: Story = {
+  args: {
+    variant: "primary",
+    inputSize: "medium",
+    placeholder: "Say something",
```

Comment:
```suggestion
    placeholder: "Email address",
```

One of our goals for these stories is to define them to be less academic and theoretical and instead to make them more practical real life situations.

Reply:
@eduramme I see this comment marked as resolved, but I don't see any change made to the placeholder value in any of the 3 "slot" related stories?

We really need you to take a step back and think about the big picture goals we need to achieve. If you don't do this then we get the problem where there's round after round after round of PR reviews and feedback to finally get the correct result. This is super time consuming for everyone.

What do the goals for the StoryBook include? Is the StoryBook supposed to just be academic and theoretical examples, or is it supposed to show useful cases that people actually want? If this wasn't already clear, 3 weeks ago I made the comment above with that specific instruction.

But now look at the actual output of all 3 of the "slot" related stories we have in Storybook. They are crap! No one wants to see this kind of output. It doesn't look sufficiently useful for the use cases of someone building an ENS app! It NEEDS to do this!!

You should be asking yourself:

1. What's a realistic example of a use case in an ENS app where someone needs to use "slot left" on an Input?
2. What's a realistic example of a use case in an ENS app where someone needs to use "slot right" on an Input?

For case 1: You've already implemented one yourself: A telegram handle input. Therefore you should update the placeholder text on slot left stories to make it look like a placeholder for a telegram username input.

For case 2: The first idea I think of is putting ".eth" or something like ".uni.eth" (given the way that the Uniswap app has built their integration with ENS). Then the placeholder text could be something like "Uniswap handle". This would then demonstrate a potentially realistic use case where some app that wants to take as input ENS names in a context where they only want names that are subnames of X (such as ".uni.eth") and integrates our `Input` component to help achieve that goal.